src/api/kube.ts

```diff
@@ -1859,12 +1859,20 @@ export class KubeHelper {
     }
   }
 
-  async clusterIssuerExists(name: string): Promise<boolean> {
+  /**
+   * Returns CRD version of Cert Manager
+   */
+  async getCertManagerK8sApiVersion(): Promise<string> {
+    const certManagerCRD = await this.getCrd('certificates.cert-manager.io')
+    return certManagerCRD.spec.version ? certManagerCRD.spec.version : CERT_MANAGER_CRD_DEFAULT_VERSION
```

Comment:
There is no `spec` without `version`,
and we can't use `CERT_MANAGER_CRD_DEFAULT_VERSION` as a default.

Reply:
I am afraid it could be possible, because `V1beta1CustomResourceDefinitionSpec` has `version` as optional field.

Reply:
I see. 
Use `versions` field and fine a version with `storage: true`. It should be the only one otherwise use `v1` as a default version.

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/

Reply:
Ok, will do